platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -10,6 +10,57 @@ const EVENTS = {
 };
 
 class HangingProtocolService {
+  studies: object[];
+  protocols: object[];
+  protocol: object;
+  stage: number;
+  _commandsManager: object;
+  ProtocolEngine: object;
+  matchDetails: object[];
+  hpAlreadyApplied: boolean[] = [];
+  customViewportSettings = [];
+  displaySets = [];
+  customAttributeRetrievalCallbacks = {
+    NumberOfStudyRelatedSeries: {
```

Comment:
Why we are hardcoding these here? These should be added by the mode/extension if it is of mode/extension interest.

Reply:
The same for `NumberOfSeriesRelatedInstances` and `ModalitiesInStudy`. It should not be part of core service, the API to add exists for extensions to add (if they want)

Reply:
They are added because they are failings of the metadata loader service to be able to load some of the standard values.